.app/postcss.js

```diff
@@ -27,8 +28,16 @@ modules.forEach(function(module) {
             console.log(`${alerts.error} ${err}`);
           } else {
             console.log(`${alerts.success} PostCss processed ${module.outDir}${module.outFile}`);
+            if(index == modules.length-1){
+              notifier.notify({
+                title: 'NYCO Patterns',
+                message: 'PostCss processing complete! 🎉'
+              });
+            }
```

Comment:
Can the notifier be optional? Perhaps passing in a parameter like `npm run development NOTIFY=true` would turn it on?